.github/workflows/release.yml

```diff
@@ -48,11 +48,17 @@ jobs:
       TURBO_TEAM: ${{ secrets.TURBO_TEAM }}
     if: github.repository_owner == 'discordjs'
     steps:
+      - uses: actions/create-github-app-token@v2
+        id: app-token
+        with:
+          app-id: ${{ vars.DISCORDJS_APP_ID }}
+          private-key: ${{ secrets.DISCORDJS_APP_KEY_RELEASE }}
+          permission-contents: write
+
       - name: Checkout repository
         uses: actions/checkout@v4
         with:
           ref: ${{ inputs.ref || '' }}
```

Comment:
We could have this use the application token too!

Reply:
Don't know if provenance supports that, we could try though?

Reply:
We can try!

Reply:
provenance appears to be unrelated to the token itself, but merely requesting the perm will set the required env vars